openlibrary/templates/home/homepage.html

```diff
@@ -0,0 +1,6 @@
+<!-- Add the slick-theme.css if you want default styling -->
```

Comment:
XXX this should be removed from this branch and added to refactor/slick-carousels